chaosgcp/dns/actions.py

```diff
@@ -0,0 +1,67 @@
+"""DNS A record update.
+
+This file has the function to update the DNS A record
+
+Typical usage example in an experiment json file:
+
+  "module": "chaosgcp.dns.actions",
+  "func": "update_service",
+"""
+
+# Copyright 2023 Google LLC.
+# SPDX-License-Identifier: Apache-2.0
+# -*- coding: utf-8 -*-
+from typing import Any, Dict
+from chaosgcp import client
+from chaoslib.types import Secrets
```

Comment:
add an extra blank line between each import blocks. Usually we follow this pattern:

```python
imports from the stdlib

import from third-parties

import from local package
```